core/errors.go

```diff
@@ -1,7 +1,23 @@
 package core
 
 import (
-	"errors"
+	"fmt"
+
+	"github.com/krystal/go-katapult"
 )
 
-var Err = errors.New("")
+//go:generate go run github.com/krystal/go-katapult/tools/codegen -t errors -p core -o . -i CoreAPI\/.* -n core/v1
```

Comment:
Does this definitely use the local version when specified like this?

Reply:
Yep, it does, cause Go is aware that the module name we're within is `github.com/krystal/go-katapult` from the `go.mod` file.

Also, if it didn't work, none of my local generation would have worked so far :)